### PR TITLE
Simpler bin/pana CLI

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: bin/pana.dart [hunk 1]
```diff
@@ -32,12 +32,9 @@ final _parser = ArgParser()
       help: 'The server that hosts <package>.', defaultsTo: defaultHostedUrl)
   ..addOption('line-length',
       abbr: 'l', help: 'The line length to use with dartfmt.')
-  ..addOption('verbosity',
-      help: 'Configure the details in the output.',
-      allowed: ['compact', 'normal', 'verbose'],
-      defaultsTo: 'normal')
-  ..addFlag('scores', help: 'Include scores in the output JSON.')
-  ..addFlag('report', help: 'Print markdown report instead of JSON.')
+  ..addFlag('hosted', help: 'Shortcut to `--source hosted`.')
```
Comment (sigurdm) [inline bin/pana.dart:35]:
```suggestion
  ..addFlag('hosted', help: 'Shortcut to `--source hosted`.', negatable = false)
```

Comment (sigurdm) [inline bin/pana.dart:35]: Can we make this one be the default? And hide the  '--source' option, just keeping it for backwards compatibility?

File: README.md [hunk 2]
```diff
@@ -37,10 +37,8 @@ Options:
       --hosted-url      The server that hosts <package>.
                         (defaults to "https://pub.dev")
   -l, --line-length     The line length to use with dartfmt.
-      --verbosity       Configure the details in the output.
-                        [compact, normal (default), verbose]
-      --[no-]scores     Include scores in the output JSON.
-      --[no-]report     Print markdown report instead of JSON.
+      --[no-]hosted     Shortcut to `--source hosted`.
+      --[no-]verbose    Print the full JSON output instead of the markdown report.
```
Comment (sigurdm) [inline README.md:41]: Should we call this `--json` or `--machine`?

Comment (isoos) [reply README.md:41]: We already have `--json` for the logs... which should be probably reused for this too (if you want json logs, why would you want non-json reports)?

**Comment by isoos** [comment]
PTAL, I have also updated/fixed the first part of the help message.

**Pull request merged**